test/unit/plugins/validate-semantic/oas3/refs.js

```diff
@@ -206,4 +206,184 @@ describe("validation plugin - semantic - oas3 refs", () => {
       return expectNoErrorsOrWarnings(spec)
     })
   })
+
+  describe("$refs for requestbody schemas must reference a schema by position", () => {
+    it("should return an error when a requestBody schema incorrectly references a local component schema", () => {
```

Comment:
**change request:** this test case's name is misleading!
```suggestion
    it("should return an error when a requestBody schema incorrectly references a local component requestBody", () => {
```

Reply:
Done!